### PR TITLE
Fix formula for module store keys in Lisk docs

### PR DESCRIPTION
**Pull request by chris529**
#1924

### CODE REVIEW

**Review by TalhaMaliktz — APPROVED** [review]
LGTM 👍🏼

File: docs/modules/ROOT/pages/understand-blockchain/state-machine.adoc [hunk 1]
```diff
@@ -61,17 +62,19 @@ A blockchain created with the Lisk SDK defines a single global store.
...
+
 Each module registered on the chain defines its own generic key-value map, where:
 
-* *key*: Module Name + Store prefix + Substore prefix + Store key
-** **Module Name**: The unique identifier of the module, identifying the key-value map.
-The module name is a unique string that is used to construct store prefix and various module-related operations.
+* *key*: Store Prefix + Substore prefix + sha256(Store key)
+
 ** **Store prefix**: The module store prefix is calculated automatically from the first 4 bytes of the hash of the module name, where the first bit of the store prefix is set to 0.
+The module name within the Lisk framework is the unique identifier of the module, identifying the key-value map.
```
Comment (Incede) [inline docs/modules/ROOT/pages/understand-blockchain/state-machine.adoc]:
```suggestion
The module name is the unique identifier of the module, identifying the key-value map.
```
Not a must but if you agree I'd omit `within the Lisk framework` as it seems a bit unnecessary and could be a bit confusing in terms of SDK vs framwework

File: docs/modules/ROOT/pages/understand-blockchain/state-machine.adoc [hunk 2]
```diff
@@ -61,17 +62,19 @@ A blockchain created with the Lisk SDK defines a single global store.
...
+It is a unique string that is used to construct store prefix and various module-related operations.
```
Comment (Incede) [inline docs/modules/ROOT/pages/understand-blockchain/state-machine.adoc]: I think module name is now not used in "various module-related operations" ?

Also the part about `used to construct store prefix` is already indicated in the previous section about "Store Prefix" and if you agree I'd omit it.
```suggestion
```

**Review by Incede — APPROVED** [review]
LGTM. I just have a minor question and 2 optional suggestions.

File: docs/modules/ROOT/pages/understand-blockchain/state-machine.adoc (hunk 2 above)
Comment (chris529) [reply docs/modules/ROOT/pages/understand-blockchain/state-machine.adoc]: @Incede Yes, good point. Thanks Rishi 👍

**Pull request merged**